src/test/java/org/folio/inventory/validation/status/GenerateItemStatusTransitionsMatrix.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.inventory.validation.status;
+
+import junitparams.JUnitParamsRunner;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RunWith(JUnitParamsRunner.class)
+public class GenerateItemStatusTransitionsMatrix {
+  private static TargetItemStatusValidators targetItemStatusValidators;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    targetItemStatusValidators = new TargetItemStatusValidators();
+  }
+
+  @Test
+  public void generate() {
+    final String fieldDelim = "\t";
+    final String stringDelim = "\"";
+    final List<String> sortedStatusNames = Arrays.stream(ItemStatusName.values()).map(itemStatusName -> itemStatusName.value()).sorted().collect(Collectors.toList());
```

Comment:
I suggest splitting this line to make it easier to identify the individual steps.

```suggestion
    final List<String> sortedStatusNames = Arrays.stream(ItemStatusName.values())
      .map(itemStatusName -> itemStatusName.value())
      .sorted()
      .collect(Collectors.toList());
```